loopchain/baseservice/peer_manager.py

```diff
@@ -110,80 +138,108 @@ def get_peer_ids_hash(peer_ids):
         util.logger.debug(f"peer ids hash({peer_ids_hash})")
         return peer_ids_hash
 
-    @property
-    def peer_list(self) -> dict:
-        """
+    def load_peers_from_iiss(self):
+        request = {
+            "method": "ise_getPRepList"
+        }
 
-        :return: { peer_id:PeerInfo }
-        """
-        # util.logger.spam(f"peer_list({self.peer_list_data.peer_info_list})")
-        return self.peer_list_data.peer_info_list
+        request = convert_params(request, ParamType.call)
+        stub = StubCollection().icon_score_stubs[ChannelProperty().name]
+        response = cast(dict, stub.sync_task().call(request))
+        response_to_json_query(response)
 
-    @property
-    def peer_order_list(self) -> dict:
-        """
+        util.logger.debug(f"in load_peers_from_iiss response({response})")
+        if 'preps' not in response['result']:
```

Comment:
If stub call failed and returned value created by `iconservice.icon_inner_service.MakeResponse.make_error_response`, returned json data may not have key named "result", which means could raise a `KeyError` exception.

Reply:
it's not written in this PR.